.github/workflows/CI - Codeql-analysis.yml

```diff
@@ -0,0 +1,67 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master, explore-site, releases/* ]
```

Comment:
Please migrate the minimum requirements into the existing FAST Validation GH Action as a step after code coverage that could be dependant on each platform build succeeding.  This way we use a single Action to validate compliance and stability-related issues once per week.  This keeps build time down for development efforts and keeps all of our actions relatively clean and well organized by function.  We also have very specific name patterns to improve readability when investigating actions.

You can keep strategy and steps without the comments and that should make it a lot less code.